tests/data/qhbm_data_test.py

```diff
@@ -17,7 +17,7 @@
 import cirq
 import tensorflow as tf
 
-from qhbmlib import quantum_data
+from qhbmlib.data import qhbm_data
```

Comment:
importing `data` doesn't work?

Reply:
It does work to import `data` and use `data.QHBMData` in the tests.  After you commented this, I checked some [TFP tests](https://github.com/tensorflow/probability/blob/main/tensorflow_probability/python/distributions/bernoulli_test.py) and saw that they use this package import style, rather than using the explicit module name.  The style of importing the module is something I was carrying over from TFQ, [see here](https://github.com/tensorflow/quantum/blob/f10d36b64761a003fd542392767fa079d57df0aa/tensorflow_quantum/python/layers/circuit_construction/elementary_test.py#L28) for example.

My thinking was that it was better to be explicit about where code is being imported from in the library tests.  But now I think I've changed my mind: with explicit module import (like `from qhbmlib.data import qhbm_data`), we also need explicit import tests to be sure the package definitions in the `__init__` files are working, [like we do here in TFQ](https://github.com/tensorflow/quantum/blob/v0.6.1/scripts/import_test.py).  If we use the package imports in the code instead, then we know the `__init__` files are working.  When we need to test things that are not in the public API, we can fall back on using module imports.  What do you think of changing to package imports throughout the code and tests?

Reply:
OK I see -- I think for tests explicit to the package it's ok to keep the finest granularity. But e.g. in `inference/qmhl_loss_test.py` it seems reasonable to import at `data` since that's the expected user usage pattern